### DISCUSSION TITLE
Release cadence

### DISCUSSION THREAD

**Discussion by m3nax** [kubernetes-sigs/gateway-api#2794] [General]
Is there a document describing the cadence of major releases?
When is the 1.1.0 release scheduled?
Is there a communication channel where I can find out information about future releases?

**Comment by robscott** [comment]
Hey @m3nax, we've historically had 2-3 releases per year, but we do not have a fixed release schedule like upstream kubernetes/kubernetes has. Generally the best place for updates on release timelines and progress is our weekly community meetings ([agenda](https://docs.google.com/document/d/1eg-YjOHaQ7UD28htdNxBR3zufebozXKyI28cl2E11tU/edit)). At this point I'm hoping we can have a release candidate for v1.1 ready sometime in March, but that may be too optimistic. At a minimum, we're aiming for v1.1 to be released in Spring 2024.

In our community meetings we are constantly tracking the set of work needed before we can release, and our overall release velocity would certainly benefit if we had more people that could help complete these tasks. You can find us in Slack or at community meetings if you're interested in helping out (https://gateway-api.sigs.k8s.io/contributing/).

**Reply by m3nax** [reply]
Thank you for the valuable information, I will take a look at the community meeting agenda as you suggested. I would like to be able to contribute to the project as I find it extremely interesting but first I need to study the GO language.
